guide.md

```diff
@@ -1543,7 +1543,7 @@ Many people have the idea that privacy-oriented services such as VPN or E-Mail p
 
 Any of those providers can be forced to silently (without your knowing (using for example a court order with a gag order[^272] or a national security letter[^273]) log your activity to de-anonymize you. There have been several recent examples of those:
 
--   2021, Proton, Proton logged IP address of French activist after an order by Swiss authorities[^274].
+-   2021, Proton, Proton logged IP address of French activist after an order by Swiss authorities (source link unavailable).

```

Comment:
I believe I found another copy of the article here: <https://techcrunch.com/2021/09/06/protonmail-logged-ip-address-of-french-activist-after-order-by-swiss-authorities/>